policy-controller/src/main.rs

```diff
@@ -159,8 +164,9 @@ async fn main() -> Result<()> {
 
     // Build the status index which will maintain information necessary for
     // updating the status field of policy resources.
-    let (updates_tx, updates_rx) = mpsc::unbounded_channel();
-    let status_index = status::Index::shared(hostname.clone(), claims.clone(), updates_tx);
+    let (updates_tx, updates_rx) = mpsc::channel(10000);
```

Comment:
Better to make this a constant to make it more visible? Also can you share the reasoning behind the number? As a point of comparison, in the endpoint translator the bounded queue size is just 100.

Reply:
queue size was picked mostly arbitrarily.  I wanted this to be big enough to hold the most patches we could possibly process in a 10 second interval (the frequency of reconciles).  If it took 1ms to apply a patch, this would be 10000.  Very conservative estimate.

Reply:
I do think that we should document this relationship. I'd also really like it if we could extract [this hardcoded 10s timeout](https://github.com/linkerd/linkerd2/blob/6e630e09fdd14e5da8d4d4db23900b9e03c55796/policy-controller/k8s/status/src/index.rs#L201) so that it's configured from the main. That way we can specify both of these constants in the same place -- ideally, up at the top of the file with the rest of the consts.